nifty-ssl/src/main/java/com/facebook/nifty/ssl/OpenSslServerConfiguration.java

```diff
@@ -61,6 +61,7 @@
         public String sessionContext = "thrift";
         public long sessionTimeoutSeconds = 86400;
         public long sessionCacheSize = 0;
+        public boolean enableStatefulSessionCache = true;
```

Comment:
You need  to also modify the `initFromConfiguration()` method, as the comment on line 58 says. Otherwise, the SslConfigFileWatcher will reset this option to the default value whenever the ticket seed file changes.